models/src/main/thrift/content/v1.thrift

```diff
@@ -377,6 +377,8 @@ struct AssetFields {
   68: optional bool safeEmbedCode
 
   69: optional bool isMandatory
+
+  70: optional string category
```

Comment:
Thanks for raising this @sophie-macmillan, 
small query for information only - what values as string we can expect in field `category`?